docker/jupyter/Dockerfile

```diff
@@ -29,7 +29,8 @@ ENV CHOWN_HOME_OPTS='-R'
 ENV REBUILD_CODE=true
 
 ARG APACHE_NEXUS=https://github.com/apache/incubator-sdap-nexus.git
-ARG APACHE_NEXUS_BRANCH=master
+ARG APACHE_NEXUS_COMMIT=be19c1d567301b09269e851cc5b5af55fea02c5d
```

Comment:
This commit is from 2019... ideally, I think we'd like this to run on the latest code? What issues did you run when running this on master?

Reply:
I checked out this commit because I was unaware that the code for nexuscli had moved (I thought it was removed) so I grabbed a commit that would've been what was used to create the old docker image. I'll retry with the current version.

Reply:
Updated to point to nexuscli in master branch. Rebuilt & tested image
`rileykkjpl/jupyter:1.5`